### ISSUE TITLE
Narrator/JAWS are not narrating name for 'Browse for folder' button present in the Data Import Dialog, while navigating using Tab/Down arrow key.

### ISSUE THREAD

**Issue by JamesFerebee** [microsoft/SqlNexus#344]
Impact on a Person with Disability (PWD):
Users who rely on screen readers will get misguided and not aware of exact information about the control, if Narrator/JAWS are not narrating name for 'Browse for folder' button present in the Data Import Dialog, while navigating using Tab/Down arrow key.

Test Environment:​
OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Tool: Narrator/JAWS (2022.2110.6).

Pre-Requisites:
1. Install SQL NEXUS application on the desktop.
2. Enable Narrator/JAWS.

Repro Steps:
Step 1: Open the SQL Nexus app in desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step 3: Navigate to Import button in left navigation and invoke it.
Step 4: Data Import Dialog gets displayed.
Step 5: Verify whether Narrator/JAWS are narrating name for 'Browse for folder' button present in the Data Import Dialog, while navigating using Tab/Down arrow key or not.
 
Actual Result:
Narrator/JAWS are not narrating name for 'Browse for folder' button present in the Data Import Dialog, while navigating using Tab/Down arrow key.
Current Observation: While navigating using focus and scan modes, Narrator is announcing as 'Button'.

Expected Result:
Narrator/JAWS Should narrate name for 'Browse for folder' button present in the Data Import Dialog, while navigating using Tab/Down arrow key.
Ex: Narrator should announce as 'Browse for folder button'.

Suggestive Fix:
Don't use scripting to transform static elements, such as a span or div to create interactive elements.
If you use scripting to create the static or interactive elements use AIRA attributes based on component or element behavior.

**Comment by JamesFerebee** [comment]
1711763

**Comment by PiJoCoder** [comment]
Fixed by PR #365
